lib/ConstExtract/ConstExtract.cpp

```diff
@@ -1326,6 +1326,25 @@ void writeConformances(llvm::json::OStream &JSON,
   });
 }
 
+void writeAllConformances(llvm::json::OStream &JSON,
+                          const NominalTypeDecl &NomTypeDecl) {
+  JSON.attributeArray("allConformances", [&] {
+    for (auto *Conformance : NomTypeDecl.getAllConformances()) {
+      auto Proto = Conformance->getProtocol();
+      // FIXME(noncopyable_generics): Should these be included?
+      if (Proto->getInvertibleProtocolKind())
+        continue;
+
+      JSON.object([&] {
+        JSON.attribute("protocol", toFullyQualifiedProtocolNameString(*Proto));
+        JSON.attribute(
+            "definingModule",
```

Comment:
Nit: Maybe `**conformance**DefiningModule` could be more apt to disambiguate from protocol-defining module.

Reply:
Makes sense. Done. I also renamed `protocol` to `protocolName` to potentially avoid reserved keyword issues in adopting clients.